Cargo.toml

```diff
@@ -15,31 +15,30 @@ members = [
 ]
 
 [dependencies]
-hex = { version = "0.4.3", default-features = false, features = ["alloc"] }
-log = { version = "0.4.14", optional = true }
-#TODO get from parity when our changes are accepted
-metadata = { version = "15.0.0", default-features = false, git = "https://github.com/integritee-network/frame-metadata", package = "frame-metadata", features = ["v14", "full_derive"] }
-#metadata = { version = "15.0.0", default-features = false, package = "frame-metadata", features = ["v14"] }
 codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ['derive'] }
+hex = { version = "0.4.3", default-features = false, features = ["alloc"] }
+log = { version = "0.4.14", default-features = false }
 primitive-types = { version = "0.12.1", optional = true, features = ["codec"] }
 serde = { version = "1.0.136", optional = true, features = ["derive"] }
 serde_json = { version = "1.0.79", optional = true }
 thiserror = { version = "1.0.30", optional = true }
 ws = { version = "0.9.2", optional = true, features = ["ssl"] }
+#TODO get from parity when our changes are accepted
+frame-metadata = { version = "15.0.0", default-features = false, git = "https://github.com/integritee-network/frame-metadata", features = ["v14", "full_derive"] }
+#frame-metadata = { version = "15.0.0", default-features = false, features = ["v14"] }
 
 # Substrate dependencies
-balances = { version = "4.0.0-dev", optional = true, git = "https://github.com/paritytech/substrate.git", package = "pallet-balances", branch = "master" }
+frame-support = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
```

Comment:
It was confusing that shortened names were used, so I removed that.

Reply:
Finally!